assets/src/components/ai/chatbot/ChatbotPanelThread.tsx

```diff
@@ -88,6 +91,18 @@ export function ChatbotPanelThread({
   })
 
   const messages = mapExistingNodes(data?.chatThread?.chats)
+
+  const [showPrompts, setShowPrompts] = useState<boolean>(isEmpty(messages))
+
+  const prompts = [
```

Comment:
can we move this to a separate file so it's easier to find quickly?

Reply:
Yes, it's done now.